spec/index.bs

```diff
@@ -935,6 +935,18 @@ and adds extra `[[`/`]]` in spec text. Consider removing the brackets.
         member corresponding to |key| in |device|.{{device/[[limits]]}} to the value |value|.
 </div>
 
+Any time a user agent wishes to revoke access to a device, it must [=lose the device=].
+
+<div algorithm>
+    To <dfn dfn>lose the device</dfn>(|device|, |reason|):
+
+    1. Set `navigator.gpu.`{{GPU/[[adapters_active]]}} to `false`.
```

Comment:
I'm not seeing where this is set back to `true`.
If I requested an adapter before, and then something happens to set this flag, and then I request a new adapter, what is it in this API that would not let me use the old adapter?

Reply:
Oops. It's supposed to happen when requestAdapter resolves.